invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides

```diff
@@ -38,13 +25,15 @@
     }
 
     td:not(.selected-member) {
-      margin-top: .75rem;
+      margin-top: 1rem;
 
-      &:before {
-        content: attr(data-label);
-        font-weight: bold;
-        margin-right: 1rem;
-        margin-left: 2rem;
+      .file-item-size {
```

Comment:
adding this as a parent selector will break all the responsive tables, no?

Reply:
You are right, I moved the styling rules to a more specific selector and reverted the changes on the more generic selector.